app/src/main/java/org/jellyfin/androidtv/ui/playback/nextup/NextUpFragment.kt

```diff
@@ -43,7 +41,11 @@ class NextUpFragment : Fragment() {
 	override fun onResume() {
 		super.onResume()
 
-		binding.fragmentNextUpButtons.startTimer()
+		if (!timerStarted) {
+			binding.fragmentNextUpButtons.startTimer()
+
+			timerStarted = true
+		}
```

Comment:
Why? The implementation of the startTimer function already restarts the timer when necessary, no need to add this check.

Reply:
This is to prevent the timer from restarting after the device comes out of a screensaver or if the device is a tv and the tv had been turned off while on the next up screen. I’m not sure it’s expected that the timer start again in those cases (or at least it wasn’t expected to me) since the user isn’t necessarily ready to start watching immediately after coming out of a screensaver/turning on the tv.

Reply:
Fair point, I think in that case you can just rename the `onResume` function to `onStart` so it only starts once.

Reply:
`onStart` is called when coming out of the screensaver/turning back (was able to reproduce this in the emulator), so I went back to using this check unless you have another suggestion

Reply:
Ok then this will do for now. Although one problem that could arise is that the activity would be destroyed in the background and is missing the value when starting again